qcodes/instrument_drivers/cryogenic/CryogenicSMS120C.py

```diff
@@ -0,0 +1,414 @@
+# Cryomagnetics_SMS120C, Cryomagnetics_SMS120C magnet power supply driver
+
+"""
+# Created on Fri 29 Nov 2017
+# @author: lyeoh
+
+# Last modified by lyeoh 27/10/2017
```

Comment:
It's kind of redundant to maintain such information in the file since we are using git.